samples/bluetooth/channel_sounding_ras_initiator/src/main.c

```diff
@@ -328,47 +328,76 @@ static void disconnected_cb(struct bt_conn *conn, uint8_t reason)
+	} else {
+		LOG_ERR("CS capability exchange failed. (HCI status 0x%02x)", status);
+	}
 }
 
-static void config_created_cb(struct bt_conn *conn, struct bt_conn_le_cs_config *config)
+static void config_create_cb(struct bt_conn *conn,
+			     uint8_t status,
+			     struct bt_conn_le_cs_config *config)
 {
-	LOG_INF("CS config creation complete. ID: %d", config->id);
-	k_sem_give(&sem_config_created);
+	ARG_UNUSED(conn);
+
+	if (status == BT_HCI_ERR_SUCCESS) {
+		LOG_INF("CS config creation complete. ID: %d", config->id);
+		k_sem_give(&sem_config_created);
+	} else {
+		LOG_ERR("CS config creation failed. (HCI status 0x%02x)", status);
+	}
 }
 
-static void security_enabled_cb(struct bt_conn *conn)
+static void security_enable_cb(struct bt_conn *conn, uint8_t status)
 {
-	LOG_INF("CS security enabled.");
-	k_sem_give(&sem_cs_security_enabled);
+	ARG_UNUSED(conn);
+
+	if (status == BT_HCI_ERR_SUCCESS) {
+		LOG_INF("CS security enabled.");
+		k_sem_give(&sem_cs_security_enabled);
+	} else {
+		LOG_ERR("CS security enable failed. (HCI status 0x%02x)", status);
+	}
 }
 
-static void procedure_enabled_cb(struct bt_conn *conn,
-				 struct bt_conn_le_cs_procedure_enable_complete *params)
+static void procedure_enable_cb(struct bt_conn *conn,
+				uint8_t status,
+				struct bt_conn_le_cs_procedure_enable_complete *params)
 {
-	if (params->state == 1) {
-		LOG_INF("CS procedures enabled:\n"
-			" - config ID: %u\n"
-			" - antenna configuration index: %u\n"
-			" - TX power: %d dbm\n"
-			" - subevent length: %u us\n"
-			" - subevents per event: %u\n"
-			" - subevent interval: %u\n"
-			" - event interval: %u\n"
-			" - procedure interval: %u\n"
-			" - procedure count: %u\n"
-			" - maximum procedure length: %u",
-			params->config_id, params->tone_antenna_config_selection,
-			params->selected_tx_power, params->subevent_len,
-			params->subevents_per_event, params->subevent_interval,
-			params->event_interval, params->procedure_interval, params->procedure_count,
-			params->max_procedure_len);
+	ARG_UNUSED(conn);
+
+	if (status == BT_HCI_ERR_SUCCESS) {
+		if (params->state == 1) {
+			LOG_INF("CS procedures enabled:\n"
+				" - config ID: %u\n"
+				" - antenna configuration index: %u\n"
+				" - TX power: %d dbm\n"
+				" - subevent length: %u us\n"
+				" - subevents per event: %u\n"
+				" - subevent interval: %u\n"
+				" - event interval: %u\n"
+				" - procedure interval: %u\n"
+				" - procedure count: %u\n"
+				" - maximum procedure length: %u",
+				params->config_id, params->tone_antenna_config_selection,
+				params->selected_tx_power, params->subevent_len,
+				params->subevents_per_event, params->subevent_interval,
+				params->event_interval, params->procedure_interval,
+				params->procedure_count, params->max_procedure_len);
+		} else {
+			LOG_INF("CS procedures disabled.");
+		}
 	} else {
-		LOG_INF("CS procedures disabled.");
+		LOG_ERR("CS procedures enable failed. (HCI status 0x%02x)", status);
```

Comment:
```suggestion
		LOG_WRN("CS procedures enable failed. (HCI status 0x%02x)", status);
```

I think LOG_WRN is more appropriate. I've seen LOG_ERR use more for fatal errors and big problems